components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -190,6 +194,20 @@ class BrowserToolbar @JvmOverloads constructor(
             editToolbar.urlView.textSize = value
         }
 
+    /**
+     * The background color used for autocomplete suggestions in edit mode.
+     */
+    var suggestionBackgroundColor: Int
+        get() = editToolbar.urlView.autoCompleteBackgroundColor
+        set(value) { editToolbar.urlView.autoCompleteBackgroundColor = value }
+
+    /**
+     * The foreground color used for autocomplete suggestions in edit mode.
+     */
+    var suggestionForegroundColor: Int?
```

Comment:
Can you explain the differences between foreground and background here? I only see one purplish color right now and want to make sure we're referencing the same thing 😄

Reply:
Sure!

Stealing images from the Android docs (since we are using `BackgroundColorSpan` and `ForegroundColorSpan` under the hood):

Background:
![image](https://user-images.githubusercontent.com/89638/52865370-cab26b00-313c-11e9-96ec-3d46849098b6.png)

Foreground:
![image](https://user-images.githubusercontent.com/89638/52865386-d0a84c00-313c-11e9-970e-c4f2e3d711d5.png)

By default the toolbar will use **the same color** for the suggestion text (foreground) as it uses for the other text. But imagine you want to have black text and also a black suggestion background. In this case you want to change the foreground/text color of the suggestion to something else.

For example this:

```
            app:browserToolbarTextColor="#ff880E4F"
            app:browserToolbarSuggestionBackgroundColor="#F50057"
            app:browserToolbarSuggestionForegroundColor="#F48FB1"
```

turns into:

![mozilla](https://user-images.githubusercontent.com/89638/52865837-c5095500-313d-11e9-9a8b-b2a40f602792.png)

Or this:

```
            app:browserToolbarTextColor="#000000"
            app:browserToolbarSuggestionBackgroundColor="#000000"
            app:browserToolbarSuggestionForegroundColor="#FFFFFF"
```

into:

![moz2](https://user-images.githubusercontent.com/89638/52865960-fbdf6b00-313d-11e9-8995-c9fe895f3435.png)